youtube_dl/downloader/http.py

```diff
@@ -45,7 +45,8 @@ class DownloadContext(dict):
             headers.update(add_headers)
 
         is_test = self.params.get('test', False)
-        chunk_size = self._TEST_FILE_SIZE if is_test else (
+        is_hls = self.params.get('hls', False)
+        chunk_size = self._TEST_FILE_SIZE if is_test and not is_hls else (
```

Comment:
No, since we don't need unnecessary traffic during tests. Basically decryption can be disabled while in test - in both cases it's technically just downloading bytes and checking whether it matches the expected checksum regardless of whether it's encrypted or not. But I'd just keep with current policy: encrypted stream should not run md5 file tests in the first place.